e2e/deploy-rook.go

```diff
@@ -97,6 +98,24 @@ func deployToolBox(c kubernetes.Interface) {
 	name := getPodName(rookNS, c, opt)
 	err := waitForPodInRunningState(name, rookNS, c, deployTimeout)
 	Expect(err).Should(BeNil())
+	waitforToolBoX(name)
+}
+
+// this is a  workaround, as we are hitting "unable to get monitor info from DNS SRV with service name: ceph-mon"
+func waitforToolBoX(name string) {
+	cmd := []string{"logs", "-nrook-ceph", name}
+	for i := 0; i < 20; i++ {
+		resp, err := framework.RunKubectl(cmd...)
+		if err != nil {
+			e2elog.Logf("failed to get logs %v", err)
+			continue
+		}
+		if !strings.Contains(resp, "=") {
```

Comment:
What is the response that should have a `=` in it? Can you give an example in a comment, or in the commit message?

Reply:
* malformed response
```
Sep  4 09:01:24.455: INFO: stdout: "Wed Sep  4 09:00:10 UTC 2019 writing mon endpoints to /etc/ceph/ceph.conf: \n"
```

* response with proper endpoint
```
Sep  4 09:01:34.655: INFO: stdout: "Wed Sep  4 09:00:10 UTC 2019 writing mon endpoints to /etc/ceph/ceph.conf: \nWed Sep  4 09:01:30 UTC 2019 writing mon endpoints to /etc/ceph/ceph.conf: a=10.96.112.168:6789\n"
```
even i can add more validation, felt  like this is enough  after lot of testing

Reply:
I didn't see  `a= ` in testing

added one more level  of validation